test/credential.policies.e2e-spec.ts

```diff
@@ -195,8 +247,8 @@ describe('Issue and Prove credentials using policies (e2e)', () => {
             .send(invitation)
             .expect((res) => {
                 expect(res.status).toBe(200);
-                expect(res.body.results.length).toBe(1);
-                expect(res.body.results[0].state).toBe('credential_acked');
+                expect(res.body.results.length).toBe(2);
```

Comment:
To be honest, I haven't liked these hardcoded quantity in the tests.  If you run the test and run it again without tearing down docker (stop, build, start), the test fails because the state is maintained between tests. Is it possible we can rewrite the test to loop through the results, look for the expected credential and test the result?  (fail if the credential isnt found or fail if the state is not as expected)?

Reply:
We could also do a greater than or equal to comparison - what do you think?

Reply:
I'll go ahead and merge this one and we can update the tests later